src/sentry/api/endpoints/organization_sessions.py

```diff
@@ -21,6 +21,17 @@
 @region_silo_endpoint
 class OrganizationSessionsEndpoint(OrganizationEventsEndpointBase):
     def get(self, request: Request, organization) -> Response:
+        query_params = MultiValueDict(request.GET)
+
+        fields = query_params.getlist("field", [])
+        anr_fields = ["anr_rate()", "foreground_anr_rate()"]
+        if any([field for field in fields if field in anr_fields]) and not features.has(
```

Comment:
```suggestion
        anr_fields = {"anr_rate()", "foreground_anr_rate()"}
        if anr_fields & set(fields) and not features.has(
```
This was a lot of `field`, maybe we can use sets to be less verbose?

Reply:
this api supports like 8 fields so that key shouldn't ever be that long, but anyway, made it sets so it's a bit more readable